emulated_hue/entertainment.py

```diff
@@ -56,7 +56,7 @@ async def async_run(self):
         # As a (temporary?) workaround we rely on the OpenSSL executable which is
         # very well supported on all platforms.
         LOGGER.info("Start HUE Entertainment Service on UDP port 2100.")
-        await self.hue.hass.async_set_state(
+        await self.hue.hass.set_state(
```

Comment:
Here, hue.hass is used while self.hass is equal to hue.hass. In other parts of the code, self.hass was removed entirely in favor of directly using hue.hass. Perhaps either remove self.hass from here as well or add self.hass back to api.py to make things more uniform?

Reply:
changed. It may not be stored as instance variable